x = 'boop'
try:
    print(int(x))
except Exception as e: # this tells what the error is
    print(e)


try:
    print(y)
except Exception as e:
    print(e)
